src/plugin/systems/rigid_body.rs

```diff
@@ -521,6 +600,17 @@ pub fn init_rigid_bodies(
             activation.angular_threshold = sleep.angular_threshold;
         }
 
+        let context_entity = entity_context_link.map_or_else(
+            || {
+                let context_entity = context.iter().next().unwrap().0;
+                commands
+                    .entity(entity)
+                    .insert(RapierContextEntityLink(context_entity));
+                context_entity
+            },
+            |link| link.0,
+        );
+        let mut context = context.get_mut(context_entity).unwrap().1;
```

Comment:
In future this should definitely have a descriptive error message saying that no available worlds exist for the entity.

Reply:
how do you feel about unwraps ? I think a user should be able to order their systems correctly to avoid these,"...

I feel like it's difficult to recover from such mistake: best case scenario is an object without rapier context, which is not updated ; not really great bug to see, arguably better than a crash, but it should fall in user mistake 🤔 ; I'll keep the crashes for now and reconsider happily depending on feedback.

Reply:
Personally I'm fine with a crash, given that a physics entity pointing to an invalid world should be an invalid state.  I just want the error message to be descriptive about what it means to avoid potential confusion about why the app crashed.